go-apps/meep-wais/server/wais.go

```diff
@@ -232,24 +265,30 @@ func updateApInfo(name string, apMacId string, longitude *float32, latitude *flo
 		}
 	}
 
+	if !needUpdate {
+		//check if AP moved
+		if oldLat != newLat || oldLong != newLong {
+			needUpdate = true
+		}
+	}
+
 	if needUpdate {
```

Comment:
It is difficult to determine why/when an update is required.
Suggest creating a function to check if update is required instead of using a flag here. Move the above code to the new function and clearly comment the logic.

Reply:
done